packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -93,7 +94,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     return (
       <div ref='root'>
         { label && (
-          <label id={ id + '-label' } className='ms-Label' ref={ (dropdownLabel) => this._dropdownLabel = dropdownLabel } >{ label }</label>
+          <Label id={ id + '-label' } ref={ (dropdownLabel) => this._dropdownLabel = dropdownLabel } required={required}>{ label }</Label>
```

Comment:
Minor nits:

The `BaseComponent` has a helper for resolving refs:

```tsx
<Label ref={ this._resolveRef('_dropdownLabel') }
```

This helper produces a callback once and then continues to return the same instance so the function isn't regenerated every time render is called.

Also nit spacing inside of the curlies in `{required}`, should be `{ required }`